blabla/sentence_processor/syntactic_feature_engine.py

```diff
@@ -43,6 +43,21 @@ def num_prepositional_phrases(const_pt):
     return len(pp_chunks)
 
 
+def num_t_units(const_pt):
+    """Returns the number of T-Units
+        Args:
+            const_pt (NLTK): The constituency parse tree
+
+        Returns
+            number of t-units (int): number of t-units
+    """
+    num_t_units = 0
+    for leaf in _leaves(const_pt, S_CLAUSE):
+        num_t_units += 1
+
+    return num_t_units
```

Comment:
I'm not familiar with how the constituency parse tree looks in Stanza/CoreNLP, but in the case of "The dog is black and she likes bones", does this correctly say 2 T-units? Or does it count the root as an `S_CLAUSE`, as well as each T-unit, and give 3?